samples/monster_generated.h

```diff
@@ -510,6 +511,14 @@ inline bool VerifyEquipment(flatbuffers::Verifier &verifier, const void *obj, Eq
   }
 }
 
+inline bool VerifyEquipmentVector(flatbuffers::Verifier &verifier, const flatbuffers::Vector<flatbuffers::Offset<void>> *values, const flatbuffers::Vector<uint8_t> *types) {
+  assert(values->size() == types->size());
```

Comment:
i'd love to return a error code / message so it's easier to debug. for now i'd just return false as you suggested. any plan/demand to add such code/msg?